packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -0,0 +1,155 @@
+import { basename, dirname, extname, join } from "path";
+import { copy, readJson, writeJson } from "fs-extra";
+import { promises } from "fs";
+import { rm } from "shelljs";
+
+export function collectDependencies(
+    externals,
```

Comment:
It is more common for rollup plugins to receive its config as a single json, should we also do that? Can make `rollup.config` more readable

Reply:
Done